src/javascript/app_2/Modules/Contract/Containers/contract-replay.jsx

```diff
@@ -0,0 +1,55 @@
+import PropType        from 'prop-types';
+import React          from 'react';
+import UILoader       from 'App/Components/Elements/ui-loader.jsx';
+import { connect }    from 'Stores/connect';
+import { isEnded }    from 'Stores/Modules/Contract/Helpers/logic';
+import Digits         from './digits.jsx';
+import InfoBox        from './info-box.jsx';
+
+class ContractReplay extends React.Component {
+    componentDidMount() {
+        this.props.onMount(this.props.contract_id);
+    }
+
+    componentWillUnmount() {
+        this.props.onUnmount();
+    }
+
+    render() {
+        const SmartChart = React.lazy(() => import(/* webpackChunkName: "smart_chart" */'../../SmartChart'));
+        const status = this.props.contract_info.status;
+
+        if (status) {
+            return (
+                <React.Suspense fallback={<UILoader />}>
+                    <SmartChart
+                        chart_id={1}
+                        is_contract_mode={true}
+                        Digits={<Digits />}
+                        InfoBox={<InfoBox />}
+                        is_ended={isEnded(this.props.contract_info)}
+                        end_epoch={this.props.contract_info.date_start}
+                        start_epoch={this.props.contract_info.date_expiry}
```

Comment:
Do we need to clear and save the chart layout here?

Reply:
@OskarAhl As per discussions with @zakibinary we've disabled `ContractReplay` for now. He will make changes to the chart layout handling for `ContractReplay` and create a PR for it. Do you think there's anything we need to change here before that?

Reply:
ok 👍 not that I can think of now.